StantUiIosLibrary/Classes/Headers/SecondHeader/SecondHeaderButton.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SecondHeaderButton.swift
+//  StantUiIosLibrary
+//
+//  Created by Mac Mini Novo on 01/08/19.
+//
+
+import UIKit
+
+public struct SecondHeaderButton {
+    let action: Selector?
+    unowned var target: UIViewController
+    let icon: UIImage?
+    
+    public init(action: Selector, target: UIViewController, icon: UIImage) {
```

Comment:
When dealing with structs, and you are going to store every named property, you don't need to specify the initialiazer because they have what it is called Memberwise Initializer. (https://stackoverflow.com/questions/38001021/why-it-is-called-the-memberwise-initialiser)

Reply:
I can't do it because this type of initializer has a maximun of an internal access control, and it struct is loccated on lib.. So it hav to be public.